askama/Cargo.toml

```diff
@@ -17,20 +17,20 @@ maintenance = { status = "actively-developed" }
 
 [features]
 default = ["config", "humansize", "num-traits", "urlencode"]
-config = ["askama_derive/config", "askama_shared/config"]
-humansize = ["askama_derive/humansize", "askama_shared/humansize"]
-markdown = ["askama_derive/markdown", "askama_shared/markdown"]
-urlencode = ["askama_derive/urlencode", "askama_shared/percent-encoding"]
-serde-json = ["askama_derive/serde-json", "askama_shared/json"]
-serde-yaml = ["askama_derive/serde-yaml", "askama_shared/yaml"]
-num-traits = ["askama_derive/num-traits", "askama_shared/num-traits"]
-with-actix-web = ["askama_derive/with-actix-web", "askama_shared/actix-web"]
-with-axum = ["askama_derive/with-axum", "askama_shared/axum"]
-with-gotham = ["askama_derive/with-gotham", "askama_shared/gotham"]
-with-mendes = ["askama_derive/with-mendes", "askama_shared/mendes"]
-with-rocket = ["askama_derive/with-rocket", "askama_shared/rocket"]
-with-tide = ["askama_derive/with-tide", "askama_shared/tide"]
-with-warp = ["askama_derive/with-warp", "askama_shared/warp"]
+config = ["askama_derive/config"]
+humansize = ["askama_derive/humansize", "dep:humansize"]
```

Comment:
This raises the MSRV for 1.60, although it seems like no MSRV than stable is enforced anyways? Otherwise could rename the dependency (and rename it back in source via `external crate humansize_cr as humansize;`).

Reply:
Even though we haven't made the MSRV explicit, I'd prefer not to consciously require 1.60 yet.